### PR TITLE
Make basictracer-py KV-compliant

### PR DESCRIPTION
**Pull request by bensigelman**

### CODE REVIEW

File: basictracer/span.py [hunk 1]
```diff
@@ -44,14 +44,30 @@ def set_tag(self, key, value):
...
+        key_values = {"event": event}
+        if payload is not None:
+            key_values["payload"] = payload
         with self._lock:
-            self.logs.append(LogData(event=event, payload=payload))
+            self.logs.append(LogData(key_values))
         return super(BasicSpan, self).log_event(event, payload)
 
     def log(self, **kwargs):
+        key_values = {}
```
Comment (yurishkuro) [inline basictracer/span.py]: I was hoping this could just use the methods in the base class.

Comment (bensigelman) [inline basictracer/span.py]: Reasonable, though the base class was doing the wrong thing :)

https://github.com/opentracing/opentracing-python/pull/37

Comment (bensigelman) [inline basictracer/span.py]: (I've pushed a change here that depends on ^^^)

**Comment by bensigelman** [comment]
OK to merge?

**Pull request merged**